fury/actor/surface.py

```diff
@@ -87,11 +91,12 @@ def surface(
         )
 
         tex = load_image_texture(texture)
-        texcoords = generate_planar_uvs(vertices, axis=texture_axis)
+        if texture_coords is None:
```

Comment:
if it is not none, is there a way to check if it is in the correct format ?

Reply:
We can just check if it is of size (num_vertices, 2)

Let me add that check